src/Compute/v2/Models/Server.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace OpenStack\Compute\v2\Models;
+
+use OpenStack\Compute\v2\Api\Server as ServerApi;
+use OpenStack\Common\Resource\AbstractResource;
+use OpenStack\Common\Resource\IsCreatableInterface;
+use OpenStack\Common\Resource\IsDeletableInterface;
+use OpenStack\Common\Resource\IsUpdateableInterface;
+use OpenStack\Common\Resource\OperatorResource;
+use OpenStack\Common\Resource\ResourceInterface;
+
+class Server extends OperatorResource implements ResourceInterface,
```

Comment:
As `OperatorResource` already implements `ResourceInterface`, does `Server` need to implement it too?